src/app.js

```diff
@@ -1,13 +1,263 @@
 // Vendor Modules
 import $ from 'jquery';
 import _ from 'underscore';
+import Trip from 'app/models/trip.js'
+import Reservation from 'app/models/reservation.js'
+import TripList from 'app/collections/tripList.js'
 
 // CSS
 import './css/foundation.css';
 import './css/style.css';
 
 console.log('it loaded!');
 
+const tripList = new TripList();
+
+const TRIP_FIELDS = [ 'id', 'name', 'continent', 'category', 'weeks','cost', 'about']
+
+const TABLE_HEADERS = [ 'name', 'continent', 'category', 'weeks','cost']
+
+const RESERVATION_FIELDS = [ 'tripID', 'name', 'age', 'email']
+
+//************************** Render ***************
+//I should not have done all the fetching in one step.  If I were to
+// go back and fix this, I would only render the "about" details if
+// a click event happened on that row.  I would get the id for the
+// fetch either by hanging it off the html somewhere (the less-good choice)
+// or figure out what Dan was talking about when he described making a closure
+// within the tripList.forEach loop that would make an event handler for each row.
+const render = function render(tripList) {
+  $('#trip-list').html('')
+  console.log(tripList);
+  tripList.forEach((trip) => {
+    let currentTrip = new Trip({id: trip.id});
+    currentTrip.fetch().done(function() {
```

Comment:
You're fetching data you don't need right now so that ends up pulling a lot of extra data and taking longer to render the list.  I suggest only doing fetch on specific trips when you need that data.